Description: Replace environment variable check with the actual value
 Instead of detecting and passing SYSTEMD_UTIL_DIR in d/rules, simply
 remove the code that checks for that variable and replace it with the
 value /lib/systemd, which should never change.
Author: Ben Westover <me@benthetechguy.net>
Forwarded: not-needed
Last-Update: 2022-09-16

--- a/src/setup.rs
+++ b/src/setup.rs
@@ -10,14 +10,7 @@
 use std::path::Path;
 use std::process::Command;
 
-pub const SYSTEMD_MAKEFS_COMMAND: &str = concat!(
-    env!(
-        "SYSTEMD_UTIL_DIR",
-        "Define $SYSTEMD_UTIL_DIR to the result of \
-         $(pkg-config --variable=systemdutildir systemd) (e.g. /usr/lib/systemd/)"
-    ),
-    "/systemd-makefs"
-);
+pub const SYSTEMD_MAKEFS_COMMAND: &str = "/lib/systemd/systemd-makefs";
 
 pub fn run_device_setup(device: Option<Device>, device_name: &str) -> Result<()> {
     let device = device.ok_or_else(|| anyhow!("Device {} not found", device_name))?;
